python/ray/serve/_private/replica.py

```diff
@@ -194,13 +198,58 @@ async def handle_request(
             *request_args,
             **request_kwargs,
         ):
-            # The request metadata should be pickled for performance.
```

Comment:
Do we still need this comment? I'm not quite sure why we `pickle` for performance here.